src/css/layout/_l-hero.scss

```diff
@@ -1,5 +1,12 @@
 .l-hero {
-	height: 75vh;
+	position: relative;
+	height: auto;
+	padding: map-get($global-bleed, small) 0;
+
+	@include mappy-bp(lap-large) {
+		height: 75vh;
+		min-height: rem(425);
```

Comment:
`min-height` to prevent header was shrinking too much on shorter viewports

Reply:
Nice.